src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -1025,11 +1025,14 @@ def update_resource_group(instance, tags=None):
 
 
 def export_group_as_template(
-        cmd, resource_group_name, include_comments=False, include_parameter_default_value=False):
+        cmd, resource_group_name, include_comments=False, include_parameter_default_value=False, resource_ids=None, skip_resource_name_params=False, skip_all_params=False):
```

Comment:
Can we register all new arguments in _params.py? Especially for `--resource-ids`, which should be an array, right?

Reply:
@Juliehzl thanks, that's what I was looking for. I noticed in the code that we have an "ids" argument and you can pass the values like:

--ids foo bar baz 

and the code will receive three values, I tried to copy the same, but when I pass multiple resource-ids in my case, I get an error saying that "foo" is not a valid argument, instead if I pass:

--resource-ids "foo bar baz" and split the string, then I get it working.

I wasn't sure what else I needed to add, could you please help me here?